src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -302,6 +308,55 @@ TEST_F(NavigationTestCaptureWindow, PanTimeWorksAsExpected) {
   PreRender();
   EXPECT_EQ(time_graph_->GetHorizontalSlider()->GetPosRatio(), 0.0);
   EXPECT_EQ(time_graph_->GetMinTimeUs(), 0.0);
+
+  // Right arrow
+  KeyPressed(20, false, false, false);
+  PreRender();
+  EXPECT_GT(time_graph_->GetHorizontalSlider()->GetPosRatio(), 0.0);
+  EXPECT_GT(time_graph_->GetMinTimeUs(), 0.0);
+
+  // Left Arrow
```

Comment:
Nit: An intermediate state would be nice, i.e. if I press "right" twice and then "left" once, I expect to be at the same position as I was after the first "right" arrow press.

Reply:
I would say that it's not always true :), and actually the test fails. I'd guess I have to zoom it a bit before.

Reply:
I included the test you suggested. After zooming-in 10 times instead of 1 we have plenty of space.